From 2b6a0e4b41a082655516c3a276a4d74e7a61cd90 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Mon, 2 Feb 2015 13:22:36 +0200
Subject: [PATCH] fix dvbs scan

using first channel freq from satellites.dat is not good.
---
 scanner.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scanner.c b/scanner.c
index 0c29ac2..25d6f8a 100644
--- a/scanner.c
+++ b/scanner.c
@@ -738,7 +738,7 @@ void cScanner::Action(void) {
                break;
              case SCAN_SATELLITE:
                aChannel->SetTransponderData(cSource::FromString(sat_list[this_channellist].source_id),
-                                          sat_list[this_channellist].items[0].intermediate_frequency,
+                                          sat_list[this_channellist].items[channel].intermediate_frequency,
                                           sat_list[this_channellist].items[channel].symbol_rate,
                                           *ParamsToString('S',
                                           GetVDRPolarizationFromDVB(sat_list[this_channellist].items[0].polarization),
-- 
1.7.10.4

